src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -490,12 +491,31 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.RM;
 		} else if (value.startsWith("theora")) {
 			format = FormatConfiguration.THEORA;
-		} else if (value.contains("windows media") || value.equals("wmv1") || value.equals("wmv2") || value.equals("wmv7") || value.equals("wmv8")) {
-			format = FormatConfiguration.WMV;
-		} else if (value.contains("mjpg") || value.contains("m-jpeg")) {
-			format = FormatConfiguration.MJPEG;
-		} else if (value.startsWith("h263") || value.startsWith("s263") || value.startsWith("u263")) {
-			format = FormatConfiguration.H263;
+		} else if (
+				value.startsWith("windows media") ||
+				value.equals("wmv1") ||
+				value.equals("wmv2")
+			) {
+				format = FormatConfiguration.WMV;
+		} else if (
+				value.contains("mjpg") ||
+				value.startsWith("mjpeg") ||
+				value.equals("mjpa") ||
+				value.equals("mjpb") ||
+				value.equals("jpeg") ||
+				value.equals("jpeg2000")
```

Comment:
With that it is, IMO, and its better than have "jpeg" displayed for video :P
```logtalk
if (media.getContainer() != null && !media.getContainer().equals(FormatConfiguration.JPG)) {
				format = FormatConfiguration.MJPEG;
			}
```
You can always believe your own eyes :tongue: 
[Moviefone_JPEG.mov.txt](https://github.com/UniversalMediaServer/UniversalMediaServer/files/1143501/Moviefone_JPEG.mov.txt)